docs/shared/register-federated-cli.mdx

```diff
@@ -1,53 +1,46 @@
-In a federated architecture, each of your graph's [implementing services](https://www.apollographql.com/docs/apollo-server/federation/implementing-services/) uses the Apollo CLI to register its schema as a separate service **of the same graph**:
+In a federated architecture, each of your graph's [subgraphs](https://www.apollographql.com/docs/federation/implementing-services/) uses the Rover CLI to register its schema with Apollo:
 
 ```mermaid
-graph BT;
+graph LR;
   subgraph " "
   gateway([Gateway]);
-  serviceA[Products service];
-  serviceB[Reviews service];
-  serviceC[Inventory service];
+  serviceA[Products subgraph];
+  serviceB[Reviews subgraph];
+  serviceC[Inventory subgraph];
   end
   registry{{Apollo Schema Registry}};
   gateway --- serviceA & serviceB & serviceC;
-  serviceA -. --serviceName=Products  .-> registry;
-  serviceB -. --serviceName=Reviews  .-> registry;
-  serviceC -. --serviceName=Inventory  .-> registry;
+  serviceA & serviceB & serviceC -. rover subgraph publish  .-> registry;
   class registry secondary;
 ```
 
-First, <a href="https://www.apollographql.com/docs/devtools/cli/" target="_blank">install the Apollo CLI</a> if you haven't yet.
+If you haven't yet:
 
-Then, **do the following for each of your implementing services**:
+*  <a href="https://www.apollographql.com/docs/rover/getting-started/" target="_blank">Install the Rover CLI.</a>
+* <a href="https://www.apollographql.com/docs/rover/configuring/" target="_blank">Authenticate Rover</a> with Apollo Studio.
 
-1. Obtain the following values, which are required for the `apollo service:push` command:
+Then, **do the following for each of your subgraphs**:
 
-    * The URL that your federated gateway will use to communicate with the service (e.g., `http://products-graphql.svc.cluster.local:4001/`).
-    * The name that uniquely identifies the service within your graph (e.g., `products`).
-    * The service's schema. The Apollo CLI can either:
+1. Obtain the following values, which are required for the `rover subgraph publish` command:
+
+    * The name that uniquely identifies the subgraph within your graph (e.g., `products`).
+        * If you've already registered a particular subgraph at least once, you can get its name from your graph's **Schema > SDL** tab in Apollo Studio, or by running `rover subgraph list`.
+        * Otherwise, you set the subgraph's name the first time you register it. **Make sure the name is unique within your graph.**
+    * The URL that your gateway will use to communicate with the subgraph (e.g., `http://products-graphql.svc.cluster.local:4001/`).
+    * The service's schema. The Rover CLI can either:
         * Use a `.gql` or `.graphql` file saved on your local machine, or
         * Perform an introspection query on the running service to fetch the schema
```

Comment:
I wonder if we should just remove these two from step 1 since we explain it in more detail in step 2. I imagine folks might come across this reading linearly and not realize they can move on to step 2 before actually getting the schema.